packages/salesforcedx-vscode-lwc/test/vscode-integration/navigation.test.ts

```diff
@@ -59,7 +53,7 @@ async function testDefinitionNavigation(
     position
   )) as Location[];
 
-  expect(locations).to.have.lengthOf(1);
+  expect(locations).to.have.lengthOf(3);
```

Comment:
why does the length change?

Reply:
The new language server allows a users to pick between the html, js or css files in a component to navigate to

Reply:
Mind adding a comment? Not obvious for people reading the code
```suggestion
  // allows a users to pick between the html, js or css files in a component to navigate to
  expect(locations).to.have.lengthOf(3);
```